bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreator.java

```diff
@@ -490,84 +361,63 @@ public class ShellyThingCreator {
+            Map.entry("shellyplussmoke", THING_TYPE_SHELLYPLUSSMOKE),
+            Map.entry("shellyplusuni", THING_TYPE_SHELLYPLUSUNI),
+            Map.entry("shellypluswdus", THING_TYPE_SHELLYPLUSDIMMERUS),
+            Map.entry("shellyplus10v", THING_TYPE_SHELLYPLUSDIMMER10V),
+            Map.entry("shelly0110dimg3", THING_TYPE_SHELLYPLUSDIMMER10V),
+            Map.entry("shellyplusrgbwpm", THING_TYPE_SHELLYPLUSRGBWPM),
+
+            // Shelly Wall Display
+            Map.entry("shellywalldisplay", THING_TYPE_SHELLYPLUSWALLDISPLAY),
+
+            // Shelly Plus Mini Series
+            Map.entry("shelly1mini", THING_TYPE_SHELLY1MINI), Map.entry("shelly1minig3", THING_TYPE_SHELLY1MINI),
+            Map.entry("shelly1minig4", THING_TYPE_SHELLY1MINI), Map.entry("shellypmmini", THING_TYPE_SHELLYPMMINI),
+            Map.entry("shellypmminig3", THING_TYPE_SHELLYPMMINI), Map.entry("shellypmminig4", THING_TYPE_SHELLYPMMINI),
+            Map.entry("shelly1pmmini", THING_TYPE_SHELLY1PMMINI),
             // Map.entry(THING_TYPE_SHELLY1PMMINIG3_STR, THING_TYPE_SHELLY1PMMINIG3),
             // Map.entry(THING_TYPE_SHELLY1PMMINIG4_STR, THING_TYPE_SHELLY1PMMINIG3),
 
-            Map.entry(THING_TYPE_SHELLYPRO1_STR, THING_TYPE_SHELLYPRO1),
-            Map.entry(THING_TYPE_SHELLYPRO1PM_STR, THING_TYPE_SHELLYPRO1PM),
-            Map.entry(THING_TYPE_SHELLYPRO1CB_STR, THING_TYPE_SHELLYPRO1CB),
-            Map.entry(THING_TYPE_SHELLYPRO2PM_RELAY_STR, THING_TYPE_SHELLYPRO2PM_RELAY),
-            Map.entry(THING_TYPE_SHELLYPRO2PM_ROLLER_STR, THING_TYPE_SHELLYPRO2PM_ROLLER),
-            Map.entry(THING_TYPE_SHELLYPRO2_RELAY_STR, THING_TYPE_SHELLYPRO2_RELAY),
-            Map.entry(THING_TYPE_SHELLYPROEM50_STR, THING_TYPE_SHELLYPROEM50),
-            Map.entry(THING_TYPE_SHELLYPRO3EM_STR, THING_TYPE_SHELLYPRO3EM),
-            Map.entry(THING_TYPE_SHELLYPRO3_STR, THING_TYPE_SHELLYPRO3),
-            Map.entry(THING_TYPE_SHELLYPRO4PM_STR, THING_TYPE_SHELLYPRO4PM),
-
-            Map.entry(THING_TYPE_SHELLYBLUBUTTON_STR, THING_TYPE_SHELLYBLUBUTTON),
-            Map.entry(THING_TYPE_SHELLYBLUDW_STR, THING_TYPE_SHELLYBLUDW),
-            Map.entry(THING_TYPE_SHELLYBLUMOTION_STR, THING_TYPE_SHELLYBLUMOTION),
-            Map.entry(THING_TYPE_SHELLYBLUHT_STR, THING_TYPE_SHELLYBLUHT),
-            Map.entry(THING_TYPE_SHELLYBLUGW_STR, THING_TYPE_SHELLYBLUGW),
-            Map.entry(THING_TYPE_SHELLYBLUGWG3_STR, THING_TYPE_SHELLYBLUGW),
+            // Shelly Pro Series
+            Map.entry("shellypro1", THING_TYPE_SHELLYPRO1), Map.entry("shellypro1pm", THING_TYPE_SHELLYPRO1PM),
+            Map.entry("shellypro1cb", THING_TYPE_SHELLYPRO1CB),
+            Map.entry("shellypro2pm-relay", THING_TYPE_SHELLYPRO2PM_RELAY),
+            Map.entry("shellypro2pm-roller", THING_TYPE_SHELLYPRO2PM_ROLLER),
+            Map.entry("shellypro2-relay", THING_TYPE_SHELLYPRO2_RELAY),
```

Comment:
This seems useless as I wouldn't expect mDNS service names to include **mode**, @markus7017? It doesn't do any harm, but now it's exposed by the changes. So if indeed this doesn't match any announced service names, we can remove them.